docs/releases/upgrading/upgrade-v10.0.md

```diff
@@ -0,0 +1,270 @@
+# Upgrading from v9 to v10
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v9, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v9, you can gradually upgrade using the existing instructions to v9 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli`, for InvenioRDM v10 the release is v1.0.6.
```

Comment:
the latest is 1.0.5, just to confirm, I assume we plan to release invenio-cli

Reply:
Yes we have to release it.